Makefile

```diff
@@ -119,8 +119,8 @@ mocks:
 	$(GOPATH)/bin/mockgen -source $(AWS_SDK_PATH)/service/acm/acmiface/interface.go -package mocks -destination ./internal/mocks/aws-sdk/acm.go
 	$(GOPATH)/bin/mockgen -source $(AWS_SDK_PATH)/service/iam/iamiface/interface.go -package mocks -destination ./internal/mocks/aws-sdk/iam.go
 	$(GOPATH)/bin/mockgen -source $(AWS_SDK_PATH)/service/route53/route53iface/interface.go -package mocks -destination ./internal/mocks/aws-sdk/route53.go
+	$(GOPATH)/bin/mockgen -source $(AWS_SDK_PATH)/service/kms/kmsiface/interface.go -package mocks -destination ./internal/mocks/aws-sdk/kms.go
```

Comment:
This is unrelated to this PR, but do we want to add a CI/CD check that runs `make mocks` and checks that there is no difference between the mocks generated and the ones in the PR? We do this in the operator for the generated code and manifests. I think it would help us catch errors when we forgot to generate mocks.

Reply:
I think it's a great idea but I may just throw a ticket in the backlog for that.

Reply:
Yep, sounds good.